src/main/java/io/github/ryotan/code/rs/inject/CodeParamConverter.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * These codes are licensed under CC0.
+ *
+ * http://creativecommons.org/publicdomain/zero/1.0/deed
+ * http://creativecommons.org/publicdomain/zero/1.0/deed.ja
+ */
+
+package io.github.ryotan.code.rs.inject;
+
+import javax.ws.rs.ext.ParamConverter;
+
+import io.github.ryotan.code.Code;
+import io.github.ryotan.code.CodeEnum;
+
+public class CodeParamConverter<C extends CodeEnum<C>> implements ParamConverter<C> {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Type Javadoc comment is missing an null tag.](https://www.codacy.com/app/ryotan/code-enum-support/file/2818755866/issues/source?bid=3375378&fileBranchId=3375797#l15)